packages/fxa-settings/src/pages/ResetPasswordWithRecoveryKeyVerified/index.tsx

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps, navigate } from '@reach/router';
+import { Localized } from '@fluent/react';
+import { logViewEvent } from '../../lib/metrics';
+import { useNavigate } from '@reach/router';
+import Ready from '../../components/Ready';
+
+type ResetPasswordWithRecoveryKeyVerifiedProps = {
+  serviceName?: string;
+};
+
+const ResetPasswordWithRecoveryKeyVerified = ({
+  serviceName,
+}: ResetPasswordWithRecoveryKeyVerifiedProps & RouteComponentProps) => {
+  const navigate = useNavigate();
+  const viewName = 'settings.reset-password-with-recovery-key-verified';
+  const baseActionName = 'reset_password_with_recovery_key_verified';
+
+  // TO-DO: Attach correct ACTUAL actions to both the link and the button
+
+  return (
+    <>
+      <Ready
+        viewName={viewName}
+        serviceName={serviceName}
+        baseActionName={baseActionName}
+      />
+      <div className="flex justify-center mx-auto m-6">
+        <Localized id="reset-password-with-recovery-key-verified-generate-new-key">
+          <button
+            className="cta-primary cta-base-p font-bold mx-2 flex-1 p-2"
+            onClick={() => {
+              const logViewName = `${viewName}.generate-new-key`;
+              const logActionName = `${baseActionName}_generate_new_key`;
+              logViewEvent(logViewName, logActionName, {
```

Comment:
Should the first argument for the logViewEvent be the viewName ('settings.reset-password-with-recovery-key-verified', and the second the eventName ('generate-new-key')?

```
export function logViewEvent(
  viewName: string,
  eventName: string,
  eventProperties: Hash<any> = {}
) {
  if (viewNamePrefix) {
    viewName = `${viewNamePrefix}.${viewName}`;
  }

  logEvents([`${viewName}.${eventName}`], eventProperties);
}
```

Example from PageDeleteAccount:
```
logViewEvent(
  'flow.settings.account-delete',
  'confirm-password.success'
);
```

Reply:
Yes! I should clarify that propname to make that more transparent -- I _thought_ I was following that pattern, but the `PageDeleteAccount` code definitely looks different 😬 . I was looking at examples like this in amplitude -- am I misunderstanding how I should do it? 
```
 'flow.save-recovery-key.copy': {
    group: GROUPS.activity,
    event: 'save_recovery_key_copy',
  },
```

Reply:
ohhhhhhh now that I'm looking at it i understand my confusion 🤦 I'll fix it!